Arrowgene.Ddon.GameServer/Quests/QuestStateManager.cs

```diff
@@ -637,6 +637,10 @@ protected PacketQueue SendWalletRewards(DdonGameServer server, GameClient client
                         var ntc = server.PPManager.AddPlayPoint(client, amount, type: 1, connectionIn: connectionIn);
                         client.Enqueue(ntc, packets);
                         break;
+                    case ExpType.AreaPoints:
+                        var areaRankNtcs = server.AreaRankManager.AddAreaPoint(client, quest.QuestAreaId, amount, connectionIn);
+                        packets.AddRange(areaRankNtcs);
```

Comment:
How come some are using Enqueue and others are using packets operations in this function?